discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -337,49 +332,19 @@
     """
 
 
-def get_latest_blocknumber(session: Session) -> Optional[int]:
-    db_block_query = (
-        session.query(Block.number).filter(Block.is_current == True).first()
-    )
-    if db_block_query is None:
-        logger.error("Unable to get latest block number")
-        return None
-    return db_block_query[0]
-
-
 def _update_aggregate_user(session):
-    most_recent_indexed_aggregate_block = get_last_indexed_checkpoint(
-        session, AGGREGATE_USER
-    )
+    current_blocknumber = get_latest_blocknumber(session)
 
-    latest_blocknumber = get_latest_blocknumber(session)
-
-    logger.info(
-        f"index_aggregate_user.py | most_recent_indexed_aggregate_block: {most_recent_indexed_aggregate_block} | latest_blocknumber: {latest_blocknumber}"
+    update_aggregate_table(
+        logger,
+        session,
+        AGGREGATE_USER,
+        UPDATE_AGGREGATE_USER_QUERY,
+        "indexed_aggregate_block",
+        current_blocknumber,
+        recalculations=True,
```

Comment:
i'll let @isaacsolo weigh in on this but we can probably drop the recalculations flag? I think that was a very specific case when the redis value gets cleared but since postgres should persist it, we should be okay?

Reply:
yea we don't need recalculation, it's ok to get rid of.

Reply:
but does this need to change at all for this PR?

Reply:
Sounds good on removing recalculations! I'll remove them.

No, aggegate_user didn't need to change, but since `update_aggregate_table()` already had metrics, figured to expose metrics for aggregate_user as well.

Reply:
Updated the PR and removed the `recalculations` parameter.